websites/B/beech/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "author": "sunraki",
+  "service": "BEECH",
+  "description": "Explore a vast collection of movies and TV shows across all genres.",
+  "url": "https://www.beech.watch",
+  "version": "1.0.0",
+  "logo": "beech",
+  "color": "#FF8800",
+  "category": "videos"
+}
```

Comment:
This metadata is invalid. Please look at the documentation for how this should be formatted